### PR TITLE
doc(focus point) :  Implement Focus Point using Gumlet

### PR DESCRIPTION
**Pull request by Athira001**
How to  implement Focus Point using Gumlet

https://github.com/quintype/ace-planning/issues/485

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test Case A
- [ ] Test Case B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tutorial/image-focus-point.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+parent: Malibu Tutorial
+nav_order: 27
+nav_exclude: true
+---
+
+# {{page.title}}
+
+_This tutorial was contributed by [Athira](https://www.linkedin.com/in/athira-m-r-835ab6105/)_
+
+You can set a focal point to ensure that a key part of an image won't get cropped. Focus points are particularly useful for responsive layouts and make sure the most important part of the image is always visible regardless of the element size. It will ensure that embarrassing image crops do not happen.
```
Comment (shraddha-kesari) [inline tutorial/image-focus-point.md:12]: Change this "and make sure the most important part of the image" to "and to  make sure the most important part of the image"

**Review by Deokumarjnu — APPROVED** [review]

File: tutorial/image-focus-point.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+## How to set image Focus Point in Bold
+
+![Image Focus Point]({{"images/set-image-focus-point.gif" | absolute_url}})
+
+After you upload an image, you can click anywhere on that image to define a Focus Point for that image. Focus Point ensures that when an image gets cropped to different sizes, the set Focus Point of the image will always be in the viewport.
```
Comment (shraddha-kesari) [inline tutorial/image-focus-point.md:18]: This "the set Focus Point of the image will always be in the viewport" can be changed to "the set Focus Point will always be in the viewport."

File: tutorial/image-focus-point.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+
+![Story API]({{"images/story-api.gif" | absolute_url}})
+
+Once you set the Focus Point, in the story API we can see the image metadata which contains the `focus-point` in an array format. 
+
+## How to implement Focus Point using Gumlet
+
+Gumlet automatically convert images to the best format and size for the user browser and device.
```
Comment (shraddha-kesari) [inline tutorial/image-focus-point.md:27]: converts

File: tutorial/image-focus-point.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+## How to implement Focus Point using Gumlet
+
+Gumlet automatically convert images to the best format and size for the user browser and device. 
+ 
+###  Extract (rect)
+
+This `extract` parameter will extract a region from an image. The `rect` is an alias for this parameter. This operation is always applied before the `resize` operation. Please give the parameters according to the original image.
+
+The value should be provided as comma-separated list `left,top,width,height`.
```
Comment (shraddha-kesari) [inline tutorial/image-focus-point.md:33]: *as a comma-separated list

File: tutorial/image-focus-point.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+We need to find the image bounds using the Image Dimensions, Aspect Ratio and Focus Point.
+
+Example: 
+
+```javascript
+function imageBounds(imageDimensions =[ 750, 422 ], aspectRatio=[ 16, 9 ], focusPoint=[ 125, 95 ]]) {
+  
+  // imageDimensions - Image Width and Height
+  // aspectRatio-  Image Aspect Ratio
```
Comment (shraddha-kesari) [inline tutorial/image-focus-point.md:53]: Add a space after aspectRatio

Comment (Athira001) [reply tutorial/image-focus-point.md:53]: done

File: tutorial/image-focus-point.md (hunk 4 above)
Comment (Athira001) [reply tutorial/image-focus-point.md:33]: done

File: tutorial/image-focus-point.md (hunk 3 above)
Comment (Athira001) [reply tutorial/image-focus-point.md:27]: done

File: tutorial/image-focus-point.md (hunk 2 above)
Comment (Athira001) [reply tutorial/image-focus-point.md:18]: done

File: tutorial/image-focus-point.md (hunk 1 above)
Comment (Athira001) [reply tutorial/image-focus-point.md:12]: done

File: tutorial/image-focus-point.md (hunk 4 above)
Comment (shraddha-kesari) [reply tutorial/image-focus-point.md:33]: Please check once on Grammarly.

**Review by shraddha-kesari — APPROVED** [review]

File: tutorial/image-focus-point.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+parent: Malibu Tutorial
+nav_order: 27
+nav_exclude: true
+---
+
+# {{page.title}}
+
+_This tutorial was contributed by [Athira](https://www.linkedin.com/in/athira-m-r-835ab6105/)_
+
+You can set a focal point to ensure that a key part of an image won't get cropped. Focus points are particularly useful for responsive layouts and to make sure the most important part of the image is always visible regardless of the element size. It will ensure that embarrassing image crops do not happen.
```
Comment (nandap4790) [inline tutorial/image-focus-point.md:12]: I think the second sentence should come first

File: tutorial/image-focus-point.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+
+## How to set image Focus Point in Bold
+
+![Image Focus Point]({{"images/set-image-focus-point.gif" | absolute_url}})
+
+After you upload an image, you can click anywhere on that image to define a Focus Point for that image. Focus Point ensures that when an image gets cropped to different sizes, the set Focus Point will always be in the viewport.
```
Comment (nandap4790) [inline tutorial/image-focus-point.md:18]: You need to mention the default focus point

**Review by nandap4790 — APPROVED** [review]

**Pull request merged**